pandas/tests/io/formats/test_to_excel.py

```diff
@@ -313,3 +314,9 @@ def test_css_to_excel_bad_colors(input_color):
     with tm.assert_produces_warning(CSSWarning):
         convert = CSSToExcelConverter()
         assert expected == convert(css)
+
+
+def tests_css_named_colors_valid():
+    upper_hexs = set(map(str.upper, string.hexdigits))
+    for color in CSSToExcelConverter.NAMED_COLORS.values():
+        assert len(color) == 6 and all(c in upper_hexs for c in color)
```

Comment:
Would you consider adding a test that all colors from ``matplotlib.colors.CS4_COLORS`` are present in ``NAMED_COLORS``?
Or would that be an overkill?

Reply:
Yeah, I wondered a bit about that, but didn't make my mind.
It's not long to write, so I've pushed a commit for that, but no problem for removing it.